src/modules/iotjs_module_buffer.c

```diff
@@ -112,26 +112,26 @@ size_t iotjs_bufferwrap_length(iotjs_bufferwrap_t* bufferwrap) {
 }
 
 
-static size_t bound_range(int index, size_t low, size_t upper) {
-  if (index < (int)low) {
+static size_t bound_range(int32_t index, size_t low, size_t upper) {
```

Comment:
First, very sorry to late reply, It was long holidays in korea. Then how about trying cast that parameter to `size_t` and and passing over?